src/org/usfirst/frc/team2335/robot/Robot.java

```diff
@@ -101,19 +136,8 @@ public void robotInit() //Runs once to initialize all global variables
 		
 		//Please for the love of god don't forget this line or else nothing works and I lose will to live
 		visionThread.start();
-		
-		driveTrain = new DriveTrain();
-		vision = new Vision();
-		
-		//This one comes last or else your code dies just like you if you don't define it last
-		oi = new OI();
-		
+	
 		cameraLight = new Relay(RELAY_PORT);
-		
-		//chooser.addDefault("Default Auto", new FindTape());		
-		//chooser.addDefault("Default Auto", new ExampleCommand());
-		// chooser.addObject("My Auto", new MyAutoCommand());
-		SmartDashboard.putData("Auto mode", chooser);
```

Comment:
Not entirely, they were moved outside of the initCamera function, because the subsystems obviously aren't a part of the camera.